### ISSUE TITLE
Several Issues (More with Isometricland than this code)

### ISSUE THREAD

**Issue by paulhoff** [mjhorvath/Video-Game-Keyboard-Diagrams#4]
I am getting the feeling that this and the Isometricland site is not maintained anymore.  Here are the issues:

- I couldn't email you directly from http://isometricland.net/email/email.php because the verification image is not showing up.
- I tried to submit a new keyboard map, but the catchpa would not  work so I could not submit.
- I tried to download the spreadsheet from https://github.com/mjhorvath/Video-Game-Keyboard-Diagrams/blob/master/sql_scripts/vgkd_bindings_template.xlsx but it appears not to exist.

I'd love to create a new keyboard map with the site, let me know if it starts working again.

Thanks,

Paul

**Comment by mjhorvath** [comment]
Thanks for the feedback. Sorry to take so long to reply.

* I fixed the email form at http://isometricland.net/email/email.php
* I tested and was not able to reproduce this
* I switched to a new method so the spreadsheet is obsolete. Unfortunately, the newer spreadsheet I created got corrupted. I suspect running LibreOffice inside a virtual machine and then suddenly closing the virtual machine may be a bad idea.
